src/LiveComponent/assets/src/morphdom.ts

```diff
@@ -41,8 +41,8 @@ export function executeMorphdom(
             }
 
             if (
-                !(fromEl instanceof HTMLElement || fromEl instanceof SVGElement) ||
-                !(toEl instanceof HTMLElement || toEl instanceof SVGElement)
+                !(fromEl instanceof SVGElement || fromEl instanceof HTMLElement) ||
+                !(toEl instanceof SVGElement || toEl instanceof HTMLElement)
```

Comment:
Did this fix a ts warning? That would be wild - it looks logically identical

Reply:
It did... but don't ask why :D My guess is that somehow the TS thinks it's pointless to check for `SVGElement` because it expects only HTMLElement. But somehow is fixed. Though I see you refactored that if without SVGElement check now :)